libs/l2/src/l2/__init__.py

```
@@ -0,0 +1,3 @@
+import requests
```

Flake8 (7.2.0)

[error] 'requests' imported but unused

(F401)